capstone/src/main/webapp/scripts/post.js

```diff
@@ -0,0 +1,48 @@
+function init() {
+  fetchBlobstoreUrlAndShowForm();
+  loadPosts();
+}
+
+function loadPosts() {
+  fetch('/post').then(response => response.json()).then((posts) => {
+    const allPostsList = document.getElementById('posts-container');
+    allPostsList.innerHTML = '';
+    for (var i = 0; i < posts.length; i++) {
+      allPostsList.appendChild(createSinglePost(posts[i]));
+    }
+  });
+}
+
+function createSinglePost(post) {
+  const postDiv = document.createElement('div');
+  postDiv.className = "post-div";
+
+  const profileImgDiv = document.createElement('div');
+  profileImgDiv.className = "post-img align-vertical";
+  postDiv.appendChild(profileImgDiv);
+
+  const postAuthor = document.createElement('h3');
+  postAuthor.className = "post-author align-vertical";
+  postAuthor.innerText = post.authorId;
+  postDiv.append(postAuthor);
+
+  const postContent = document.createElement('p');
+  postContent.className = "post-content";
+  postContent.innerText = post.postText;
+  postDiv.append(postContent);
```

Comment:
You are doing a good job here with naming, but could you break each of these into their own helper functions that return the constructed element? That would help with readability and not needing to worry about knowing how a postAuthor etc. is created. I know you still need the constructed elements to add to postDiv, so you can do something like
`postDiv.append(createAuthor(post))`

Reply:
sounds good, I created helped functions for each component

Reply:
Will also make sure to make separate and smaller PR for different features!